CHANGELOG.md

```diff
@@ -12,26 +12,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ## [2.0.0](https://github.com/burzum/cakephp-file-storage/releases/tag/2.0.0) - 2018-09-07
+### Changed
+- Added this change log
+- Improved documentation
+
+## [2.0.0-rc1](https://github.com/burzum/cakephp-file-storage/releases/tag/2.0.0-rc1) - 2018-09-07
+### Changed
+- Updated dependencies
+
+## [2.0.0-beta2](https://github.com/burzum/cakephp-file-storage/releases/tag/2.0.0-beta2) - 2018-09-07
+### Changed
+- Improved documentation
+- Updated dependencies
+- Upgraded to CakePHP 3.6
+- Removed upload validation methods as they are part of the CakePHP core
+- Increased the length of the extension field in the DB #157
+- Updated Travis CI configuration, remove PHP 5.6, add PHP 7.1, 7.2 & nightly
+
+### Fixed
+- Fixed Travis CI builds
+
+## [2.0.0-beta1](https://github.com/burzum/cakephp-file-storage/releases/tag/2.0.0-beta1) - 2017-11-25
 ### Added
 - Added Flysystem support to StorageManager
 - Added pre- and post processing callbacks for image processing
 
 ### Changed
-- Upgraded to CakePHP 3.6
```

Comment:
Why was this line removed?

Reply:
It was moved to https://github.com/burzum/cakephp-file-storage/pull/185/files/7026176bce2c8b8eadfe240a9ee3c8a15d79dc81#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR27